activestorage/test/models/preview_test.rb

```diff
@@ -70,6 +70,61 @@ class ActiveStorage::PreviewTest < ActiveSupport::TestCase
     assert_changes -> { !!preview.image.variant(transformations)&.send(:processed?) }, to: true do
       preview.processed
     end
+
+    image = read_image(preview.image.variant(transformations))
+    assert_equal 216, image.width
+    assert_equal 280, image.height
```

Comment:
I don't think these assertions are necessary for this particular test.  This test only cares about whether the variant was processed.  Tests in `variant_test.rb` should check whether `Variant#processed` works as expected.

Reply:
I agree